tests/havannah-state-tests.cc

```diff
@@ -208,7 +218,7 @@ TEST(HavannahStateGroup, init_2) {
   ASSERT_EQ(0, action.GetX());
   ASSERT_EQ(i, action.GetY());
   ASSERT_EQ(j, action.GetZ());
-  ASSERT_EQ(h, action.GetHash());
+  ASSERT_EQ(0, action.GetHash()); // Niki changed to 0
```

Comment:
Is the hash always 0? What is this testing? Maybe you should just remove `getHash` entirely.

Reply:
Yeah before merging this, as we already changed tests for it in this PR

Reply:
Ok, I have done this